src/borg/archiver.py

```diff
@@ -1999,17 +2128,17 @@ def process_epilog(epilog):
         `repokey` and `keyfile` use AES-CTR-256 for encryption and HMAC-SHA256 for
         authentication in an encrypt-then-MAC (EtM) construction. The chunk ID hash
         is HMAC-SHA256 as well (with a separate key).
-        These modes are compatible with borg 1.0.x.
+        These modes are compatible with Borg 1.0.x.
```

Comment:
not sure if we had set a policy like "always write 'Borg' as the short form" and also this might be inconsistent at a thousand other places, so maybe this rather needs such a policy and a global cleanup outside of this PR.